### DISCUSSION TITLE
500+ vulns after running ansible-role-rhel8-stig

### DISCUSSION THREAD

**Discussion by svbfromnl** [ComplianceAsCode/content#8652] [Q&A]
Hi! New guy here. What am I doing wrong? I have a new rhel8 (8.5) AWS ec2 instance that I'm trying to STIG using the RH ansible playbook https://github.com/RedHatOfficial/ansible-role-rhel8-stig
After running, successfully, a Nessus scan using v1.5 of the DISA RHEL STIG generates over 500 vulns. 
Did I miss something?

ETA: whoops, I just read that the ansible playbook is based on v1.4 of the STIG. Can't delete this post, so let me see if I can find v1.4 and try that.

**Comment by konstruktoid** [comment]
Could you please define the 500 vulns?

**Reply by svbfromnl** [reply]
I think the issue is with Tenable sc. I noticed it flagged vulns that aren't actually on the system. Even simple things like the root ownership of /var/log/messages, which clearly is owned by root when I look at it. Let me sort that out first.

**Comment by marcusburghardt** [comment]
Please, reopen the discussion if still relevant.
